util/getopt.h

```diff
@@ -40,6 +40,17 @@
 	getopt_initloop_ ## x:
 #define DO_LONGJMP							\
 	goto *getopt_initloop
+
+/* Work around GCC < 9 bug. */
```

Comment:
Is this a GCC bug, or a Solaris bug?  Isn't the problem that Solaris isn't defining sigsetjmp properly?